content/tracing/languages/php.md

```diff
@@ -217,7 +217,12 @@ dd_trace("CustomDriver", "doWork", function (...$args) {
 
 ## Configuration
 
-Configure the Agent connection parameters via environment variables.
+The PHP tracer can be configured using environment variables.
+
+*An important note*: If you use code auto-instrumentation (the recommended approach) please be aware that the
```

Comment:
In order to avoid duplication, would it be better to shorten this section to one sentence with a link to the env advanced docs? :)

Reply:
Fix the same as above if this is kept

Reply:
@SammyK if this makes sense to you, I preferred to move the details to this PHP configuration section (generic) and link to this section from the advanced tracing page (more specific to the debug context).